app/assets/javascripts/models/beta/story.js

```diff
@@ -142,13 +147,28 @@ export const toggleStory = (story) => {
 };
 
 const stateFor = (story) => {
-  if (story.state === "unscheduled" && typeof story.estimate === "number") return "unstarted";
-  if (story.state === "unstarted" && isUnestimatedFeature(story)) return "unscheduled";
-  if (!isFeature(story) && story.state === "unscheduled") return "unstarted";
+  if (story.state === status.UNSCHEDULED && story.estimate) return status.UNSTARTED;
+  if (story.state === status.UNSTARTED && isUnestimatedFeature(story)) return status.UNSCHEDULED;
+  if (!isFeature(story) && story.state === status.UNSCHEDULED) return status.UNSTARTED;
   return story.state;
 }
 
+const isEstimable = (story) => {
+  return story.storyType === storyTypes.FEATURE
```

Comment:
We can use this function https://github.com/Codeminer42/cm42-central/blob/2f09b8958e2a32a1b3cc523ae5063d81594b612b/app/assets/javascripts/models/beta/story.js#L43 here :smile: